CHANGES

```diff
@@ -7,6 +7,19 @@ PyVISA Changelog
 No changes yet
 
 
+1.9.1 (unreleased)
+-----------------
+
+- Add expect_termination argument to read_binary_values and query_binary_values.
+  For some instruments, this retains the behavior of these methods from pyvisa
+  1.8, where the length of the block of values that is being read did not
+  include the very final read termination. For those instruments with pyvisa
+  1.9.0, visa library raises a timeout exception. The behavior is not reverted
+  completely (to its pyvisa 1.8 state) because there are claims that some
```

Comment:
Could you make that statement less dismissive now that I have provided a real world example ?

Reply:
sure